lib/user-interface/react-app/src/pages/rag/create-workspace/create-workspace-opensearch.tsx

```diff
@@ -15,9 +15,10 @@ const nameRegex = /^[\w+_-]+$/;
 const defaults: OpenSearchWorkspaceCreateInput = {
   name: "",
   embeddingsModel: null,
+  crossEncodingEnabled: false,
```

Comment:
default value should reflect the value from the configuration

Reply:
Here we are just defining the constant, the value from the configuration is read below at line 38.